dowhy/causal_refuters/partial_linear_sensitivity_analyzer.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, estimator=None, num_splits=5, shuffle_data=False,
                  shuffle_random_seed=None,  reisz_polynomial_max_degree=3,
                  r2yu_tw=0.04, r2tu_w=0.03, significance_level=0.05, benchmark_common_causes=None,
                  frac_strength_treatment=None, frac_strength_outcome=None,
-                 observed_common_causes=None, treatment=None, outcome=None, g_s_estimator_list=None, alpha_s_param_dict=None, g_s_estimator_param_list=None):
+                 observed_common_causes=None, treatment=None, outcome=None, g_s_estimator_list=None, alpha_s_param_dict=None, g_s_estimator_param_list=None , **kwargs):

```

Comment:
since this class does not use alpha_s_param_dict, it should not explicitly written here. It will go in kwargs when the nonparametric class is called

Reply:
It does use alpha_s_param_dict for calculating alpha_s_j. (alpha after removing benchmark covariates)